internal/proto/local/controller/api/resources/hostsets/v1/host_set.proto

```diff
@@ -48,9 +48,21 @@ message HostSet {
 	repeated string host_ids = 100 [json_name="host_ids"];
 
 	// An ordered list of endpoint preferences used to choose from among
-	// multiple possible endpoints for a host.
+	// multiple possible endpoints for a host. Preferences are specified by
+	// "cidr:<valid IPv4/6 CIDR>" or "dns:<globbed name>", specifying which IP
+	// address or DNS name out of a host's available possibilities should be
+	// preferred. If no preferences are specified, a value will be chosen _at
+	// random_ from all available values. May not be valid for all plugin types.
 	repeated string preferred_endpoints = 101 [json_name="preferred_endpoints", (custom_options.v1.generate_sdk_option) = true];
 
+	// An interger number of seconds indicating the amount of time that should
+	// elapse between syncs of the host set. The interval will be applied to the
+	// end of the previous sync operation, not the start. Setting to any
+	// negative value will disable syncing for that host set; setting to zero
+	// (or null) will cause the set to use Boundary's default. The default may
```

Comment:
When we discussed I think we decided on not allowing zero and requiring the standard used elsewhere of setting it to null to use the default.

Reply:
Correct, I need to update this.